uui-core/src/types/tables.ts

```diff
@@ -225,6 +225,8 @@ type FilterConfigBase<TFilter> = {
     columnKey: string;
     isAlwaysVisible?: boolean;
     predicates?: IFilterPredicate[];
+    /** default value: 2. count of words to show in the Filter PickerToggler */
```

Comment:
Move info about default to the end of the description:
`/** Count of words to show in the Filter toggler. By default 2 item will be shown. */`

Reply:
fixed